extension/slack.js

```diff
@@ -0,0 +1,1192 @@
+const plt = { platform: "slack" };
```

Comment:
What is a Slack meeting? 

Are you trying to build this for a Slack Huddle? Else the implementation seems a bit off, because there is not really an official Slack meeting as such